libraries/SrcWrapper/src/stm32/hw_config.c

```diff
@@ -19,13 +19,37 @@
 extern "C" {
 #endif
 
+/**
+  * @brief  This function enables clocks for some system IP
+  * @param  None
+  * @retval None
+  */
+void hw_config_system_IP_clock_init(void)
```

Comment:
Wonder if this function should not be moved to clock.c ?

Reply:
Moved and renamed